lib/bindings/MQTTBinding.js

```diff
@@ -110,7 +110,7 @@ function generateCommandExecution(apiKey, device, attribute) {
     var options = {};
     payload = ulParser.createCommandPayload(device, cmdName, cmdAttributes);
     if (config.getConfig().mqtt.qos) {
-        options.qos = config.getConfig().mqtt.qos;
+        options.qos = parseInt(config.getConfig().mqtt.qos) || 0;
```

Comment:
Different from IOTA-JSON (https://github.com/telefonicaid/iotagent-json/pull/324/) the check appears only one time in the IOTA-UL case. Is this correct? (just to be sure)

Reply:
Yes, iota-ul doesn't implements  sendConfigurationToDevice